### ISSUE TITLE
Lycoris extension slow down iteration by 200% without being used

### ISSUE THREAD

**Issue by LoliStalker** [KohakuBlueleaf/a1111-sd-webui-lycoris#67]
Disclaimer I'm new to Stable diffusion and just started last week

Long story short:
Somehow having Lycoris extension cause my GPU to not be able to perform fully and I have no idea why, when I enable the extension despite not using it and it just being there it drops my generation time by 500%. It went back to normal after I disable the extension. It sucks that I can't use Lycoris function but it isn't the end of the world, still would be cool if I could use it and if someone experience something similar and found the fix for it, please share!

edit: My GPU is 1060, windows 10 
// version: [v1.7.0] •  python: 3.10.6  •  torch: 2.0.1+cu118  •  xformers: 0.0.20  •  gradio: 3.41.2  •  

Full story:
So anyway I was trying all of the features of stable diffusion and yesterday I installed the Lycoris extension and I haven't used it yet because I'm still trying out other features, I noticed that my generation time increases after I turn on my PC this morning and my iteration time goes by 100-200% from 1-2s/it to 5-8s/it and this happens to all settings. Txt2img, img2img, etc with or without hires.fix and at high or low sampling steps which confuse me greatly so I scramble to try to fix it on my own.

The first thing I noticed was when I checked on my Task manager, my GPU Cuda usage usually at 90-100% goes to 40-60% and it wasn't static, and yet the performance goes beyond half, it would make sense if the GPU usage drop cuz of temperature or CPU throttling etc that the result should somewhat follow the performance drop so I expect the time to double but nope, it was more than quadruple! Usually, it took me 30 seconds or less to generate a basic 20-step image without additional settings but now it become 3 minutes!

I tried googling it and I found a lot of people with similar issues that suddenly their stable diffusion slowing down mostly due to updates but no one ever mentioned their Cuda usage in their GPU at all which confused me since it was the first thing I notices, so I thought it was common issue but rarely do I find any positive result other than rolling back webui version and that doesn't work for me so ugh

I tried to restart my PC multiple time after installing and uninstalling my GPU driver, using Malwarebytes and so on and so forth but I finally took a step back and reminded myself what changes since yesterday that I added to my PC/setting, and the only thing that come to mind was Lycoris extension so I try to disable the extension and it fix everything.

**Comment by KohakuBlueleaf** [comment]
<img width="752" alt="image" src="https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris/assets/59680068/33cf8c49-331c-4884-ac19-3e97d2920ffd">

**Comment by KohakuBlueleaf** [comment]
<img width="117" alt="image" src="https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris/assets/59680068/7ea935e5-b97c-4412-8235-c8ea5802f0d2">

**Comment by LoliStalker** [comment]
Thank you, that explain it. Is there any way to use Lycoris in v1.7? This is the only lycoris I found or is the extension dead?

**Comment by KohakuBlueleaf** [comment]
> Thank you, that explain it. Is there any way to use Lycoris in v1.7? This is the only lycoris I found or is the extension dead?

Built-in lora system is mainly maintained by me after sd-webui 1.5.0

Now guess how you use lycoris model in sd-webui 1.7

UwU

**Comment by LoliStalker** [comment]
"Image Generation
[a1111/sd-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
After sd-webui 1.5.0, LyCORIS models are officially supported by the built-in LoRA system. You can put them in either models/Lora or models/LyCORIS and use the default syntax <lora:filename:multiplier> to trigger it.

When we add new model types, we will always make sure they can be used with the newest version of sd-webui.

As for sd-webui with version < 1.5.0, please check this [extension](https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris)."

So basically I don't need the extension? Oh well, my bad for using an old guide, these things change and progress too quickly. Anyway thank you for the response, I was so lost

**Comment by KohakuBlueleaf** [comment]
Yes
Just use \<lora:...:...\>
The arg system is as same as this extension
